gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/ProductDependencyIntrospectionPlugin.java

```diff
@@ -112,7 +112,8 @@ private static String getMinimumProductVersion(Project project, String group, St
                 .map(dependency -> project.getDependencies().create(ImmutableMap.of(
                         "group", dependency.getProductGroup(),
                         "name", dependency.getProductName(),
-                        "version", dependency.getMinimumVersion())))
+                        "version", dependency.getMinimumVersion(),
+                        "ext", "pom")))
```

Comment:
These keys are defined in https://github.com/gradle/gradle/blob/v5.4.0/subprojects/dependency-management/src/main/java/org/gradle/api/internal/notations/DependencyMapNotationConverter.java#L45

Reply:
I don't think this works in all cases and doesn't leave room for an escape hatch. For example, most published assets do not produce a POM and therefore can not be resolved in this manner